src/SearchTest.java

```diff
@@ -0,0 +1,30 @@
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+public class SearchTest
+{
+	Search search;
+	int arr[]={-10,-2,0,1,5,8,15};
+	int to_search=5;
+	@Before
+	public void initializeClass()
+	{
+		search=new Search();
+	}
+	@Test
+	public void linearTest()
+	{
+		assertEquals(4,search.linearSearch(arr,to_search));
+	}
+	@Test
+	public void binaryTest()
+	{
+		assertEquals(4,search.binarySearch(arr,0,arr.length,to_search));
+	}
```

Comment:
Test at-least one positive and one negative flow for each method

Reply:
done

Reply:
Still missing test method for case when key not present into the Array. And this is what i mean by negative flow test in above comment.

Reply:
added test case for negative test case, i.e element not in array